torch/_inductor/comms.py

```diff
@@ -124,28 +130,64 @@ class ReorderInfo:
     final_exposed: float = -1
     limiting_factor: str = "None"
     moves: int = 0
+    grouped: int = 0
+    grouped_info: str = ""
 
     @property
     def improvement(self):
         return self.initial_exposed - self.final_exposed
 
 
+def is_gemm_like(node: Optional[Union[IRNode, Operation]]) -> bool:
```

Comment:
is "gemm-like" here a proxy for "has meaningful flops?" (which would include other fallback ops like e.g. custom matmul triton kernels from users).

I wonder if there are any other inductor utils to tell us this info with higher fidelity... Maybe @eellison will have ideas when he's back